SVS/model/network.py

```diff
@@ -813,7 +813,7 @@ def __init__(
         self.use_asr_post = use_asr_post
         self.d_model = d_model
 
-    def forward(self, phone, pitch, beats):
+    def forward(self, phone, pitch, beats, args):
```

Comment:
you only use vocoder_category for the function, there is no need to pass the whole args to the function